src/Api/Persistent.hs

```diff
@@ -88,3 +100,56 @@ findUserByLogin login = find (\u -> u ^. userLogin == login) <$> getUsers
 -- | FIXME: anyone can login
 loginUser :: ST -> Persist ()
 loginUser login = modifyDb dbCurrentUser . const . fmap (view _Id) =<< findUserByLogin login
+
+-------------------------------------------------------------------
+-- HACK to make easy to emulate db savings from prototypes
+-- FIXME: This is not part of the interface
+
+forceLogin :: Integer -> Persist ()
+forceLogin x = modifyDb dbCurrentUser (const (Just (AUID x)))
+
+nextId :: Persist (AUID a)
+nextId = do
+    modifyDb dbLastId (+1)
+    AUID <$> getDb dbLastId
+
+currentUser :: Persist (AUID User)
+currentUser = fromJust <$> getDb dbCurrentUser
```

Comment:
I prefer` (\(Just v) -> v)`, as it gives a better error message and a compile-time non-totality warning if -Wall is enabled.